prow/cmd/mkpj/BUILD.bazel

```diff
@@ -9,8 +9,14 @@ load(
 
 go_binary(
     name = "mkpj",
-    args = ["--config-path=$(location //prow:config.yaml)"],
-    data = ["//prow:config.yaml"],
+    args = [
+        "--config-path=$(location //prow:config.yaml)",
+        "--job-config-path=config/jobs",
```

Comment:
Have you tried this? Does it work as expected? I would expect this to also have a $(location) part

Reply:
it works locally at least.